README.md

```diff
@@ -24,6 +24,6 @@ To build the Calico Networking Plugin for CNI locally, clone this repository and
 
 [cni]: https://github.com/appc/cni
 [config]: configuration.md
-[calico-containers]: https://github.com/projectcalico/calico-containers/blob/master/docs/cni/kubernetes/README.md
+[calico-containers]: https://github.com/projectcalico/calico-containers/blob/master/README.md
```

Comment:
I think we actually want to link to docs.projectcalico.org instead - we've moved all the docs and guides over there, so calico-containers doesn't have it anymore.